_schemaV2.graphql

```diff
@@ -1162,27 +1162,33 @@ type AnalyticsVisitorsByReferral {
   value: Int!
 }
 
-# App Authenticator Two-Factor Authentication factor
```

Comment:
This diff should be safe for clients - `yarn relay` results in zero changes with this diff.